src/main/java/com/rackspace/salus/policy/manage/services/TenantManagement.java

```diff
@@ -76,37 +77,54 @@ public String getAccountTypeByTenant(String tenantId) {
     return metadata.get().getAccountType();
   }
 
-  /**
-   * Create or update the information stored relating to an individual tenant.
+  /**ˍ
+   * Update the information stored relating to an individual tenant.
    * @param tenantId The tenant to store this data under.
    * @param input The data to alter.
    * @return The full tenant information.
    */
-  public TenantMetadata upsertTenantMetadata(String tenantId, TenantMetadataCU input) {
-    Optional<TenantMetadata> metadata = tenantMetadataRepository.findByTenantId(tenantId);
+  public TenantMetadata updateMetaData(String tenantId, TenantMetadataCU input) {
+    log.info("Updating tenant metadata for {}", tenantId);
 
-    TenantMetadata updated;
-    if (metadata.isEmpty()) {
-      log.info("Creating tenant metadata for {}", tenantId);
-      updated = new TenantMetadata()
-        .setTenantId(tenantId);
-    } else {
-      log.info("Updating tenant metadata for {}", tenantId);
-      updated = metadata.get();
+    TenantMetadata metadata = getMetadata(tenantId).orElseGet(() -> {
+      return new TenantMetadata().setTenantId(tenantId);
+    });
+
+    return upsertTenantMetaData(tenantId, input, metadata);
+  }
+
+  /**
+   * Create the information for an individual tenant
+   * @param tenantId
+   * @param input
+   * @return The full tenant information
+   */
+  public TenantMetadata createMetaData(String tenantId, TenantMetadataCU input) {
```

Comment:
```suggestion
  public TenantMetadata createMetadata(String tenantId, TenantMetadataCU input) {
```

Should have caught this first time around. Metadata is one word.